flow/Platform.cpp

```diff
@@ -518,7 +518,7 @@ const char* getInterfaceName(const IPAddress& _ip) {
 		if(!iter->ifa_addr)
 			continue;
 		if (iter->ifa_addr->sa_family == AF_INET && _ip.isV4()) {
-			uint32_t ip = ntohl(((struct sockaddr_in*)iter->ifa_addr)->sin_addr.s_addr);
+			uint32_t ip = ntohl((((struct sockaddr_in*)iter->ifa_addr)->sin_addr.s_addr));
```

Comment:
Why is extra parenthesis needed here?

Reply:
Because clang probably warns here. However, I would prefer if we don't use c-style casts...

```suggestion
			uint32_t ip = ntohl(((reinterpret_cast<struct sockaddr_in*>(iter)->ifa_addr)->sin_addr.s_addr));
```

Reply:
Switched out C-style cast to more favorable type